src/FsAutoComplete.Core/XmlDoc.fs

```diff
@@ -0,0 +1,963 @@
+module FsAutoComplete.XmlDoc
```

Comment:
this was all moved from TipFormatter because I think we might need it in more locations. unsure though.